.pipelines/android_packaging.yml

```diff
@@ -0,0 +1,113 @@
+        displayName: Start Android emulator
+        workingDirectory: $(OnnxruntimeDirectory)
+
+      - task: JavaToolInstaller@0
+        displayName: Use jdk 11
+        inputs:
+          versionSpec: "11"
+          jdkArchitectureOption: "x64"
+          jdkSourceOption: "PreInstalled"
+
+      - template: templates/set-package-version-variable-step.yml
+        parameters:
+          IsReleaseBuild: ${{ parameters.IsReleaseBuild }}
+          PackageVersionVariableName: ORT_EXTENSIONS_AAR_VERSION
+
+      - script: |
+          python ./tools/gen_selectedops.py ./tools/android/package_ops.config
+        displayName: "Generate selected ops CMake file"
+
+      - script: |
+          bash ./tools/android.package.sh
+        displayName: Build onnxruntime-extensions aar package
+        workingDirectory: $(Build.sourcesdirectory)
+
+      - script: |
+          cd out/android/Release/java/androidtest/android
+
+          ./gradlew --no-daemon clean connectedDebugAndroidTest
+        displayName: Run extensions unit test with Android Emulator
+        workingDirectory: $(Build.sourcesdirectory)
+
+      - script: |
+          set -e -x
+          cd out/android/Release/java/androidtest/android
+
+          appcenter test run espresso \
+          --app "AI-Frameworks/Ort-Ext-Mobile-Android-Testapp" \
+          --devices $(app_center_android_test_devices) \
+          --app-path ./app/build/outputs/apk/debug/app-debug.apk \
+          --test-series "master" \
+          --locale "en_US" \
+          --build-dir ./app/build/outputs/apk/androidTest/debug \
+          --token $(app_center_api_token)
+        displayName: Run extensions Android unit test using App Center
+        workingDirectory: $(Build.sourcesdirectory)
+
+      - script: |
+          set -e -x
+
+          AAR_STAGING_DIR="$(Build.sourcesdirectory)/out/android/Release/java/build/android/outputs/aar"
+          LIBS_STAGING_DIR="$(Build.sourcesdirectory)/out/android/Release/java/build/android/libs"
+          POM_FILE_STAGING_DIR="$(Build.sourcesdirectory)/java/build/publications/maven"
+          ARTIFACTS_STAGING_DIR="$(Build.ArtifactStagingDirectory)"
+
+          PROJECT_NAME="onnxruntime-extensions"
+          PACKAGE_NAME="onnxruntime-mobile-extensions"
+          VERSION_FILE="$(Build.SourcesDirectory)/version.txt"
+          PROJECT_VERSION="$(cat "${VERSION_FILE}")"
+          cp ${LIBS_STAGING_DIR}/${PROJECT_NAME}-${PROJECT_VERSION}-javadoc.jar  ${ARTIFACTS_STAGING_DIR}
+          cp ${LIBS_STAGING_DIR}/${PROJECT_NAME}-${PROJECT_VERSION}-sources.jar  ${ARTIFACTS_STAGING_DIR}
+          cp ${AAR_STAGING_DIR}/${PROJECT_NAME}-release.aar                      ${ARTIFACTS_STAGING_DIR}
+          cp ${POM_FILE_STAGING_DIR}/pom-default.xml                             ${ARTIFACTS_STAGING_DIR}
+          mv ${ARTIFACTS_STAGING_DIR}/${PROJECT_NAME}-${PROJECT_VERSION}-javadoc.jar  ${ARTIFACTS_STAGING_DIR}/${PACKAGE_NAME}-${ORT_EXTENSIONS_AAR_VERSION}-javadoc.jar
```

Comment:
is it sufficient to rename the files with a new version number? do we need to update build-android.gradle's project.version too?
https://github.com/microsoft/onnxruntime-extensions/blob/81e7799c69044c745239202085eb0a98f102937b/java/build-android.gradle#L24

Reply:
yeah it would be better if we can directly generate the aar file with dev/release version number, but I am not sure how to replace the ../version.txt here with the package version we generated in the pipeline (base and dev version + hash suffix)

Reply:
one idea is to read a value from an environment variable if it is defined and otherwise use the contents of version.txt. something like
`project.version = System.getenv('ORT_EXTENSIONS_AAR_VERSION') ?: rootProject.file('../version.txt').text.trim()`